kolibri/content/test/test_content_app.py

```diff
@@ -74,7 +74,7 @@ def test_get_top_level_topics(self):
 
         p = content.ContentNode.objects.get(title="root")
         expected_output = content.ContentNode.objects.filter(parent=p, kind=content_kinds.TOPIC)
-        actual_output = content.ContentNode.objects.get(parent__isnull=True).get_children().filter(kind=content_kinds.TOPIC)
```

Comment:
We don't allow the parent to be null anymore?

Reply:
Oh, I guess it's just a test.

Reply:
Parent can be null, it's just that's not completely disambiguating, as we could have multiple root nodes in a DB, so better to be safe than sorry in our test writing :)